public/app/percona/inventory/Inventory.test.tsx

```diff
@@ -53,6 +53,46 @@ describe('Inventory tables', () => {
     expect(container.querySelectorAll('tr')).toHaveLength(5);
   });
 
+  it('Agents table only first page', () => {
+    const response = {
+      pmm_agent: [{ agent_id: 'pmm-server', runs_on_node_id: 'pmm-server', connected: true }],
+      node_exporter: new Array(50).fill({
+        agent_id: '/agent_id/262189d8-e10f-41c2-b0ae-73cc76be6968',
+        pmm_agent_id: 'pmm-server',
+        status: 'RUNNING',
+        listen_port: 42000,
+      }),
+      postgres_exporter: new Array(50).fill({
+        agent_id: '/agent_id/8b74c54e-4307-4a10-9a6f-1646215cbe07',
+        pmm_agent_id: 'pmm-server',
+        service_id: '/service_id/ab477624-4ee9-49cd-8bd4-9bf3b91628b2',
+        username: 'pmm-managed',
+        status: 'RUNNING',
+        listen_port: 42001,
+      }),
+
+      qan_postgresql_pgstatements_agent: new Array(50).fill({
+        agent_id: '/agent_id/ac55153c-5211-4072-a5de-59eb2a136a5c',
+        pmm_agent_id: 'pmm-server',
+        service_id: '/service_id/ab477624-4ee9-49cd-8bd4-9bf3b91628b2',
+        username: 'pmm-managed',
+        status: 'RUNNING',
+      }),
+    };
+
+    const { container } = render(
+      <Table
+        data={InventoryDataService.getAgentModel(response as any)}
```

Comment:
aren't we missing props to make sure only 25 show up, like `showPagination` ?

Reply:
Yup, this one looks wrong, I'll fix it.

Reply:
On second look, the test section is weird, I'll update it.